Makefile

```diff
@@ -88,6 +88,12 @@ check_database:
 # en erna \dt
 # om te zien of migration is gelukt
 
+# run prisma studio on port 5555
+studio:
+		@export DATABASE_URL="postgresql://Transcendancingqueens:8:uizdY5._r-Pe+@localhost:5432/pongdb?schema=public"
```

Comment:
we should not forget to remove this at some point, because it contains our private login details that should only be in .env and should never be pushed..

Reply:
very sharp of you mr. Flen!
i will put it into our TO DO BEFORE SuBMITTING list on our notion page.